lib/rules/selector-pseudo-class-no-unknown/index.js

```diff
@@ -17,7 +17,8 @@ const { isString } = require('../../utils/validateTypes');
 const ruleName = 'selector-pseudo-class-no-unknown';
 
 const messages = ruleMessages(ruleName, {
-	rejected: (selector) => `Unexpected unknown pseudo-class selector "${selector}"`,
+	rejected: (/** @type {any} */ selector) =>
+		`Unexpected unknown pseudo-class selector "${selector}"`,
```

Comment:
[suggest] This addition of `/** @type {any} */` seems needless. Is it possible to remove it?

Reply:
@ybiquitous I have a error without it.

```sh
lib/rules/selector-pseudo-element-no-unknown/index.js:17:13 - error TS7006: Parameter 'selector' implicitly has an 'any' type.

17  rejected: (selector) => `Unexpected unknown pseudo-element selector "${selector}"`,

Reply:
Can you try clean-installing `node_modules/` via `npm ci`?

Reply:
Done.